models/_sector/dex/trades/base/platforms/smardex_base_base_trades.sql

```diff
@@ -0,0 +1,21 @@
+{{
+    config(
+        schema = 'smardex_base',
+        alias = 'base_trades',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['tx_hash', 'evt_index'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{{
+    smardex_compatible_trades(
+        blockchain = 'base',
+        project = 'smardex',
+        version = '1',
+        Pair_evt_Swap = source('smardex_base', 'SmardexPair_evt_Swap'),
+        Factory_evt_PairCreated = source('smardex_base', 'SmardexFactory_evt_PairCreated')
+    )
+}}
```

Comment:
We may use `uniswap_compatible_v3_trades` here:
```suggestion
{{
    uniswap_compatible_v3_trades(
        blockchain = 'base',
        project = 'smardex',
        version = '1',
        Pair_evt_Swap = source('smardex_base', 'SmardexPair_evt_Swap'),
        Factory_evt_PoolCreated = source('smardex_base', 'SmardexFactory_evt_PairCreated'),
        pair_column_name = pair
    )
}}
```